frontend/packages/dev-console/src/components/pipelineruns/PipelineTaskStatus.tsx

```diff
@@ -1,34 +1,39 @@
 import * as React from 'react';
-import { VictoryStack, VictoryBar } from 'victory';
-import { Tooltip } from '@console/internal/components/utils/tooltip';
-import {
-  getTaskStatus,
-  PipelineRun,
-  runStatus,
-  getRunStatusColor,
-} from '../../utils/pipeline-augment';
-import TaskStatusToolTip from './TaskStatusTooltip';
+import { Firehose } from '@console/internal/components/utils';
+import { PipelineRun } from '../../utils/pipeline-augment';
+import { PipelineModel } from '../../models';
+import { PipelineBars } from './PipelineBars';
 
 interface PipelineTaskStatusProps {
   pipelinerun: PipelineRun;
+  pipeline?: any;
```

Comment:
Do we have a type for a pipeline instead of using `any`?

Reply:
Updated to `K8sKind`